src/Samples/TestZipkin.cs

```diff
@@ -55,14 +55,13 @@ private static void DoWork(int i)
             // 6. Get the global singleton Tracer object
             ITracer tracer = Tracing.Tracer;
 
+            ISpan span = null;
             // 7. Start another span. If another span was already started, it'll use that span as the parent span.
             // In this example, the main method already started a span, so that'll be the parent span, and this will be
             // a child span.
-            using (OpenCensus.Common.IScope scope = tracer.SpanBuilder("DoWork").StartScopedSpan())
+            using (OpenCensus.Common.IScope scope = tracer.SpanBuilder("DoWork").StartScopedSpan(out span))
```

Comment:
you can define `span` right inside the `out`. It is a new syntax in C#. Something like this looks much better:

``` csharp
using (var scope = tracer.SpanBuilder("DoWork").StartScopedSpan(out var span))
```

Reply:
so you wouldn't need globally scoped `ISpan span = null;`

Reply:
Done, Thanks !